content/en/docs/reference/issues-security/official-cve-feed.md

```diff
@@ -3,7 +3,8 @@ title: Official CVE Feed
 weight: 25
 outputs:
```

Comment:
```suggestion
linkTitle: CVE feed
feedTitle: Kubernetes CVE Feed
outputs:
```
and then use that `feedTitle`, not “Official CVE Feed”, as the feed titles (JSON Feed, RSS). Otherwise people might not know whose official feed they're considering subscribing to.

Reply:
I'm not sure how to access the page frontmatter variables from the layout files. I added the `linkTitle` however :)!

Reply:
See https://gohugo.io/content-management/front-matter/#user-defined

Reply:
Yep it works, but do you think it's a good idea to replace the real title "Auto-refreshing Official CVE Feed" to  "Kubernetes CVE Feed" in the JSON and XML feeds?

cc @PushkarJ

Reply:
I think several projects might plausibly use a title like “Auto-refreshing Official CVE Feed” for a feed, and endusers would find it useful to tell these apart.

Reply:
I recommend following to be used everywhere 
> Auto-refreshing Official Kubernetes CVE Feed"

Reply:
This gives us the best of both options

Reply:
>Auto-refreshing Official Kubernetes CVE Feed

It's a long name. However, something for another PR. If we deliver RSS, this one is done.

Reply:
Ok to do the name change in another PR 💯

Reply:
If you want to lgtm or approve this one it would be charming 😇